Update session-host-update.md (#125823)
Azure/azure-content: articles/virtual-desktop/session-host-update.md

Here are known issues and limitations:

> - The current session host configuration version is not displayed in the session host blade in the portal. Until it is fixed, you can access the portal using this URL to view the current configuration version: https://portal.azure.com/?feature.hostpoolVirtualMachinesBladeV3=false#home

Reviewer: ```suggestion
- The current session host configuration version isn't displayed in the session host blade in the portal. Until this issue is fixed, you can access the portal using [this portal URL](https://portal.azure.com/?feature.hostpoolVirtualMachinesBladeV3=false#home) to view the current configuration version.
```
